presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
Tempto already has yaml merge logic, since the "local" config file overrides some of the settings in the global yaml file.  So the best solution is probably to have tempto accept multiple values for "--config-local"